components/ShopHeader/index.js

```diff
@@ -14,9 +14,14 @@ export default function ShopHeader({ cartTotal }) {
   return (
     <Styled.ShopHeader>
       <Styled.ShopH1>Shop Name</Styled.ShopH1>
-      <Styled.ShoppingCartLink href={"/ShoppingCart"}>
+      <Styled.ShoppingCartLink
+        aria-label="Checkout link"
+        href={"/ShoppingCart"}
+      >
         <SmallCart />
-        <span>{cartTotal}</span>
+        {cartTotal !== 0 ? (
```

Comment:
```suggestion
        {cartTotal ? (
```

?

Reply:
- [x] Kind of makes more sense, thanks. Fixed.